src/plugins/notebook/components/NotebookMenuSwitcher.vue

```diff
@@ -50,66 +57,53 @@ export default {
     },
     mounted() {
         validateNotebookStorageObject();
-        this.getDefaultNotebookObject();
 
         this.notebookSnapshot = new Snapshot(this.openmct);
         this.setDefaultNotebookStatus();
     },
     methods: {
-        getDefaultNotebookObject() {
-            const defaultNotebook = getDefaultNotebook();
+        getPreviewObjectLink() {
+            const getRelativePath = this.openmct.objects.getRelativePath(this.objectPath);
```

Comment:
I think the variable name should be just `relativePath` rather than `getRelativePath`?